perf/src/sigverify.rs

```diff
@@ -471,10 +471,11 @@ impl Deduper {
         }
     }
 
+    // Returns 1 if packet is to be discarded. Else, 0.
     fn dedup_packet(&self, packet: &mut Packet) -> u64 {
         // If this packet was already marked as discard, drop it
         if packet.meta.discard() {
-            return 0;
+            return 1;
```

Comment:
Just going by the name of the class and function, if I were to write a unit test to just count how many packets are duplicate, it'll fail if some packets were marked as discard.

Maybe we should rename the function or move the logic to count discarded packets out of this function. What do you think?

Reply:
I like the idea of having separate functions strictly for dedup vs. counting discarded packets, but I'm worried about doing anything to increase latency in SigVerify (having to take another loop through all of the packets), so I'm leaning towards a rename